e2e-tests/playwright/tests/visual/channels/intro_channel.spec.ts

```diff
@@ -17,10 +17,10 @@ test('Intro to channel as regular user', async ({pw, pages, browserName, viewpor
     await channelsPage.toBeVisible();
 
     // Wait for Boards' bot image to be loaded
-    await pw.shouldHaveFeatureFlag('OnboardingAutoShowLinkedBoard', true);
-    const boardsWelcomePost = await channelsPage.getFirstPost();
-    await expect(await boardsWelcomePost.getProfileImage('boards')).toBeVisible();
-    await wait(duration.one_sec);
+    // await pw.shouldHaveFeatureFlag('OnboardingAutoShowLinkedBoard', true);
+    // const boardsWelcomePost = await channelsPage.getFirstPost();
+    // await expect(await boardsWelcomePost.getProfileImage('boards')).toBeVisible();
+    // await wait(duration.one_sec);
```

Comment:
If would be nice to have a way to condition this test by checking the feature flag value instead of commenting it out

Reply:
Maybe adding a helper function ie. `isFeatureFlagEnabled`, so if/else cases can be tested.
https://github.com/mattermost/mattermost-server/blob/master/e2e-tests/playwright/support/flag.ts

Reply:
for the purpose of getting this merge asap I will let this test commented out and will open a new PR creating a new test for covering both scenarios